frontend/public/components/dashboard/dashboard-card/card-help.tsx

```diff
@@ -10,8 +10,8 @@ export const DashboardCardHelp: React.FC<DashboardCardTitleHelpProps> = React.me
     <OverlayTrigger overlay={overlay} placement="top" trigger={['click']} rootClose>
       <Button bsStyle="link">
         <Icon
-          type="fa"
-          name="info-circle"
+          type="pf"
+          name="icon-info"
```

Comment:
It should actually be without `icon`, like this:

```suggestion
          name="info"
```

applies for others too. Sorry for confusing you earlier :)